### PR TITLE
[Alerta] Do not set status, fix rawData and default severity is indeterminate

### PR DESCRIPTION
**Pull request by satterly**
Some minor fixes to the Alerta integration...

* let the Alerta server determine the alert status otherwise alerts that have been acknowledged in the console will be incorrectly modified
* change incorrect `data` attribute to `rawData`
* default severity should be `indeterminate` otherwise automatic status changes don't work as expected

- [x] CHANGELOG.md updated
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](http://influxdb.com/community/cla.html) (if not already signed)

### CODE REVIEW

**Comment by nathanielc** [comment]
@satterly Thanks for the PR, can you help me understand how Alerta determines the status?

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Example UDF config for a socket based UDF.
 - [#441](https://github.com/influxdata/kapacitor/issues/441): Fix panic in UDF code.
 - [#429](https://github.com/influxdata/kapacitor/issues/429): BREAKING: Change TICKscript parser to be left-associative on equal precedence operators. For example previously this statement `(1+2-3*4/5)` was evaluated as `(1+(2-(3*(4/5))))`
     which is not the typical/expected behavior. Now using left-associative parsing the statement is evaluated as `((1+2)-((3*4)/5))`.
+- n/a: Fixes Alerta integration to let server set status, fix `rawData` attribute and set default severity to `indeterminate`.
```
Comment (nathanielc) [inline CHANGELOG.md]: Now that you have on open PR can you reference the PR here in the CHANGELOG?

**Comment by satterly** [comment]
@nathanielc See http://docs.alerta.io/en/latest/server.html#state-based-browser

**Comment by nathanielc** [comment]
@satterly Perfect. Thanks.

**Pull request merged**